README.md

```diff
@@ -42,51 +42,59 @@ Your code interview will be evaluated based on your repo, so make sure all files
 
 ## Introduction
 
+The challenge has two halves, a python/bash half with 4 steps and a docker/SQL half with 2 steps. The second half will use a file generated by the first python step. 
+
+### Python/bash expectations
+The logic to perform each step should be written in python, and be callable by an executable script that takes the arguments.  
+
 We love the NYC 311 service and the open data products that come with it. In this challenge, you will use **[311 Service Requests from 2010 to Present](https://data.cityofnewyork.us/Social-Services/311-Service-Requests-from-2010-to-Present/erm2-nwe9)** on NYC Open Data, write an ETL pipeline, and produce some data insight.
 
-### Task 1: Data Download
 
-Write a python script/notebook to download all service request records created in the **last week** (7 days) and has **HPD** as the responding agency, and store the data in a csv named `raw.csv` in a folder called `data`.
+### Python Task 1: Data Download
+
+Write a process to download all service request records created in the last *x* days with a certain responding agency. It should take the number of days and responding agency acronym as parameters passed from the command line. Save the service requests in the last seven days with HPD as the responding agnecy and a second different pull of the data in two adequatly named csv files in a folder called `data`.
 
-### Task 2: Data Aggregation
+### Python Task 2: Data Aggregation
 
-Create a time series table based on the `data/raw.csv` file we created from **Task 1** that has the following fields
+Write a process to produce a time series table based on the data you've pulled so far. It should have the following fields
 
-- `created_date_hour`: the timestap of request creation by date and hour
-- `complaint_type`: the type of the complaint
+- `created_time`: the timestap of request creation by date and hour OR just date
+- `complaint_type`: the type of the complaint (optional)
 - `count`: the count of service requests by `complaint_type` by `created_date_hour`
 
-Store this table in a csv under the `data` folder with a csv file name of your choice.
+The granularity of the `created_time` column and whether the data is broken down by complaint type should be controlled by parameters passed in from the command line. 
```

Comment:
This needs to be moved up top, to explicitly state the you should be able to pass parameters through the process.